src/theme.js

```diff
@@ -84,6 +85,11 @@ export const HeaderLinkImage = styled.img`
 
   @media (max-width: ${smallSize}) {
     height: 80px;
+    width: 50vw;
+  }
+
+  @media (max-width: ${xSmallSize}) {
+    width: 40vw;
```

Comment:
maybe it is better to let it fill the page? Because then logo squishes disproportionately

```
flex: 1;
margin: 0 1rem 0 1rem;
```

![image](https://user-images.githubusercontent.com/1270272/136463732-17a22d2c-6ea0-4635-b59d-b0ac9b0257be.png)

Reply:
Thanks for the contribution though!